roles/node-access-checks/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Ping nodes from deployment host
+  ping:
+
+- name: Ping nodes from deployment host, and 'become'
+  ping:
+  become: true
+
+- name: Assert remote SSH user is not 'root'
```

Comment:
I'm not getting what's the purpose of all of this. For me #337 did it right

Reply:
Not necessarily: I didn't test, but

- When running `ansible-playbook` as `root`, `ansible_user` wouldn't be set and you'd nonetheless be SSH'ing as `root`
- When having some fancy configuration in `~/.ssh/config` to specify a specific user for a specific host

so I thought we're better off adding some actual dynamic checks as well to be (even more) safe.

Reply:
I think having another solution:
```
- setup:
  become: False
- assert:
       that:
            - "ansible_user_id != 'root'"
- setup: # Restablish default setup with info that only root might be able to get